protos/deployment.proto

```diff
@@ -64,6 +66,15 @@ message DeploymentSpec {
       string function_auth_level = 5;
   }
 
+  message AwsEc2OperatorConfig {
+    string region=1;
+    string instance_type=2;
+    string ami_id=6;
+    int32 autoscale_min_capacity=7;
```

Comment:
suggest renaming `min_size`, `max_size`, make it consistent with the native API so it's easier for users to reason about its behavior in the cloud platform

Reply:
Since all three here are parameters of an autoscaling-group, maybe just add `autoscale` to its original parameter name like the following?
```
autoscale_min_size
autoscale_max_size
autoscale_desired_capacity
```

Reply:
Do you also want to change it from cli,operator,uitls and constants to avoid confusion?

Reply:
yes please also change it in CLI and gRPC API to avoid confusion.